### PR TITLE
Fixed numbered bullets

### PR DESCRIPTION
**Pull request by AnthonyHakim**

### CODE REVIEW

File: CenturyLink Private Cloud on VMware Cloud Foundation/ip-sec-dnat.md [hunk 1]
```diff
@@ -14,57 +14,57 @@ In this KB article, we walk through how to enable an IPSec VPN on Edge Gateway S
...
+*	Add IPsec VPN:
+  * Enabled: Click slider to enable
+  * Enable perfect forward secrecy (PFS): default
+  * Name: Name your IPSec VPN
+  * Local Id: Your Local Id
+  * Local Endpoint: IP address of Edge Gateway
+  * Local Subnets: Your Local Subnets
+  * Peer Id: Your Peer Id
+  * Peer Endpoint: IP address of Peer
+	* Peer Subnets: Your Peer Subnets
```
Comment (jim-phillips) [inline CenturyLink Private Cloud on VMware Cloud Foundation/ip-sec-dnat.md]: fix indent

**Comment by jim-phillips** [comment]
@AnthonyHakim 1 comment, and please resolve conflicts.

**Comment by AnthonyHakim** [comment]
Sorry @jim-phillips, I will get this fixed.

**Comment by jim-phillips** [comment]
@AnthonyHakim you still have conflicts.

**Comment by jim-phillips** [comment]
@AnthonyHakim to redo.

**Pull request closed without merging**